test/performance/perf_test.lua

```diff
@@ -314,6 +568,16 @@ local pairs_perf = {
 }
```

Comment:
It is not obvious from the final report that the testing is performed using different parameters (fiber/connection count) for different rows. I would show the parameters somewhere.

Reply:
Added